Polyhedron/demo/Polyhedron/Plugins/Classification/Classification_plugin.cpp

```diff
@@ -835,7 +835,7 @@ public Q_SLOTS:
       if (label_buttons.size() > 1)
         for (int i = position + 1; i < label_buttons.size(); ++ i)
         {
-          int position = i - 1;
+          int position = static_cast<int>(i - 1);
```

Comment:
This does not fix any warning. `i` is already an `int` and the expression `i - 1` is also an `int`.

The warning was about the comparison in the condition of the `for`-loop:
```
/Users/cgaltester/cgal_test/CGAL-4.12-Ic-59/cmake/platforms/x86-64_Darwin-13.0_Apple-clang-5.0_Release/test/Polyhedron_Demo/Plugins/Classification/Classification_plugin.cpp:836:38: warning: comparison of integers of different signs: 'int' and 'size_type' (aka 'unsigned long') [-Wsign-compare]
        for (int i = position + 1; i < label_buttons.size(); ++ i)
                                   ~ ^ ~~~~~~~~~~~~~~~~~~~~
1 warning generated.
```

Reply:
Damn, you're right… sorry about that, I'll fix it.